### PR TITLE
SEO-176113-Xamarin-Page-With-Redirect

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the links with redirects.

Regards,
Beryl.

### CODE REVIEW

File: Xamarin/Expander/Appearance.md [hunk 1]
```diff
@@ -133,6 +133,6 @@ N> When effects view is applied to `Expander` by adding Visual as `Material`, it
...
-[How to use a custom font icon for Expander in Xamarin.Forms (SfExpander)](https://www.syncfusion.com/kb/11469/)
+[How to customize header icon in Xamarin.Forms Expander (SfExpander)](https://support.syncfusion.com/kb/article/10051/how-to-customize-header-icon-in-xamarin-forms-expander-sfexpander)                                                                                                                                                                                                         
+[How to center align the Expander header text in Xamarin.Forms (SfExpander)](https://support.syncfusion.com/kb/article/9908/how-to-center-align-the-expander-header-text-in-xamarin-forms-sfexpander?isIn)
```
Comment (Yvone-Atieno) [inline Xamarin/Expander/Appearance.md:137]: Check this kb link

File: Xamarin/Installation/install-nuget-packages.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ documentation: ug
 
 ## Overview
 
-**NuGet** is a Package management system for Visual Studio. It makes it easy to add, update and remove external libraries in our application. Syncfusion publishing all Xamarin  NuGet packages in [nuget.org](https://www.nuget.org/packages?q=Tags%3A%22xamarin%22+syncfusion). The Syncfusion Xamarin NuGet packages can be used without installing the Syncfusion Essential Studio setup. To create with the Syncfusion Xamarin components, just use the Syncfusion Xamarin NuGet packages in a Xamarin application.
+**NuGet** is a Package management system for Visual Studio. It makes it easy to add, update and remove external libraries in our application. Syncfusion publishing all Xamarin  NuGet packages in [nuget.org](https://www.nuget.org/packages?q=Tags:%22xamarin%22+syncfusion). The Syncfusion Xamarin NuGet packages can be used without installing the Syncfusion Essential Studio setup. To create with the Syncfusion Xamarin components, just use the Syncfusion Xamarin NuGet packages in a Xamarin application.
```
Comment (Yvone-Atieno) [inline Xamarin/Installation/install-nuget-packages.md:15]: Check this link if it's redirecting

File: Xamarin/Installation/install-nuget-packages.md [hunk 3]
```diff
@@ -12,9 +12,9 @@ documentation: ug
...
 
-> From v16.2.0.46 (2018 Volume 2 Service Pack 1) onwards, all the Syncfusion Xamarin components are available as NuGet packages at [nuget.org](https://www.nuget.org/packages?q=Tags%3A%22xamarin%22+syncfusion).
+> From v16.2.0.46 (2018 Volume 2 Service Pack 1) onwards, all the Syncfusion Xamarin components are available as NuGet packages at [nuget.org](https://www.nuget.org/packages?q=Tags:%22xamarin%22+syncfusion).
```
Comment (Yvone-Atieno) [inline Xamarin/Installation/install-nuget-packages.md:17]: Check this link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Expander/Appearance.md (hunk 1 above)
Comment (berylouma) [reply Xamarin/Expander/Appearance.md:137]: Corrected

File: Xamarin/Installation/install-nuget-packages.md (hunk 2 above)
Comment (berylouma) [reply Xamarin/Installation/install-nuget-packages.md:15]: Updated the correct link

File: Xamarin/Installation/install-nuget-packages.md (hunk 3 above)
Comment (berylouma) [reply Xamarin/Installation/install-nuget-packages.md]: Corrected

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request merged**